src/DIPS.Xamarin.UI/Controls/TrendGraph/TrendGraph.xaml.cs

```diff
@@ -0,0 +1,233 @@
+        {
+            get { return (Color)GetValue(GraphBackgroundColorProperty); }
+            set { SetValue(GraphBackgroundColorProperty, value); }
+        }
+
+        /// <summary>
+        ///  <see cref="MaxValue" />
+        /// </summary>
+        public static readonly BindableProperty MaxValueProperty =
+            BindableProperty.Create(nameof(MaxValue), typeof(double), typeof(TrendGraph), 100.0, propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// Max value of the trend
+        /// </summary>
+        public double MaxValue
+        {
+            get { return (double)GetValue(MaxValueProperty); }
+            set { SetValue(MaxValueProperty, value); }
+        }
+
+        /// <summary>
+        ///  <see cref="MinValue" />
+        /// </summary>
+        public static readonly BindableProperty MinValueProperty =
+            BindableProperty.Create(nameof(MinValue), typeof(double), typeof(TrendGraph), 0.0, propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// Min value of the trend
+        /// </summary>
+        public double MinValue
+        {
+            get { return (double)GetValue(MinValueProperty); }
+            set { SetValue(MinValueProperty, value); }
+        }
+
+        /// <summary>
+        ///  <see cref="GraphMargin" />
+        /// </summary>
+        public static readonly BindableProperty GraphMarginProperty =
+            BindableProperty.Create(nameof(GraphMargin), typeof(double), typeof(TrendGraph), 0.5, propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// Margin between each item
+        /// </summary>
+        public double GraphMargin
+        {
+            get { return (double)GetValue(GraphMarginProperty); }
+            set { SetValue(GraphMarginProperty, value); }
+        }
+
+        /// <summary>
+        ///  <see cref="ItemsSource" />
+        /// </summary>
+        public static readonly BindableProperty ItemsSourceProperty =
+            BindableProperty.Create(nameof(ItemsSource), typeof(IList), typeof(TrendGraph), propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// Items used to draw the trends. Use <see cref="ValueMemberPath" /> to expose path to item values
+        /// </summary>
+        public IList ItemsSource
+        {
+            get { return (IList)GetValue(ItemsSourceProperty); }
+            set { SetValue(ItemsSourceProperty, value); }
+        }
+
+        /// <summary>
+        ///  <see cref="ValueMemberPath" />
+        /// </summary>
+        public static readonly BindableProperty ValueMemberPathProperty =
+            BindableProperty.Create(nameof(ValueMemberPath), typeof(string), typeof(TrendGraph), string.Empty, propertyChanged: OnAnyPropertyChanged);
+
+        /// <summary>
+        /// To expose path to item values. Set this if the items are not of type int, double or float.
+        /// </summary>
+        public string ValueMemberPath
```

Comment:
The property path to use as a value. The value is used to determine how the height of the graph.